pkg/storj/path.go

```diff
@@ -4,6 +4,10 @@
 package storj
 
 import (
+	"github.com/skyrings/skyring-common/tools/uuid"
```

Comment:
import ordering

Reply:
We already had uuid imported. I just moved it

Reply:
Not in `storj` package. It's the thing that everything imports, so avoiding deps there is somewhat important for binary sizes.

Reply:
moved back to metainfo